app/filtering.js

```diff
@@ -692,9 +694,35 @@ const initPartition = (partition) => {
     options.parent_partition = ''
   }
   if (isTorPartition) {
+    // TODO(riastradh): Duplicate logic in app/browser/tabs.js.
```

Comment:
it is actually a bug @diracdeltas mentioned in https://github.com/brave/muon/pull/473#issuecomment-376620108

Reply:
@riastradh-brave can you split this out into a separate module so it's not in here twice? for instance you could add it as an exported method in `app/channel.js`